Essentials/src/messages_en.properties

```diff
@@ -19,6 +19,10 @@ antiBuildInteract=\u00a74You are not permitted to interact with\u00a7c {0}\u00a7
 antiBuildPlace=\u00a74You are not permitted to place\u00a7c {0} \u00a74here.
 antiBuildUse=\u00a74You are not permitted to use\u00a7c {0}\u00a74.
 autoAfkKickReason=You have been kicked for idling more than {0} minutes.
+autoTeleportDisabled=\u00a76Automatic teleportation request approval \u00a7cdisabled\u00a76.
```

Comment:
I think `"You are now automatically approving teleport requests."` and `"You are no longer automatically approving teleport requests."` might be better - opinions?

Also, should we warn users if they *enable* automatic teleport requests while having teleportation disabled?

Reply:
Yeah I think that sounds better. I was just trying to force "enabled" and "disabled" in the wording, since that seems to be the norm for toggle commands.

Instead of a warning, perhaps it should automatically enable teleportation and notify you. In most  cases the player probably wants to do this anyway. Although, I could see people arguing either way on this.

Reply:
> Although, I could see people arguing either way on this.

I think changing it automatically could annoy players. Server admins can add aliases to turn it on/off if they want, I guess.

Alternatively it could be a config option, but then that itself might confuse players when it doesn't behave consistently across different servers.

Reply:
Yeah It's pretty harmless but I definitely see how it could be annoying (in cases where the player is just testing the command or for some reason still wants teleportation disabled). I think there's already enough config options as-is and this probably doesn't need to be another one. It's probably just fine to handle the case where teleportation is disabled by printing an additional message like "Note that players will not be able to teleport to you until you have enabled teleportation".